### ISSUE TITLE
XMRig static Linux executable compilation issue

### ISSUE THREAD

**Issue by matmad** [xmrig/xmrig#3016]
I have a problem with compilation of XMRig into static Linux executable. When I compile it using instructions given in the documentation I get "Segmentation fault (core dumped)" on fresh Linode (Ubuntu 21.10) installation. I am compiling on Kali. At the same time static executable, that I can download from official XMRig website works perfect on fresh Linode. I have tried compiling using Alpine docker and compiled executable worked on the docker and on Kali machine, but does not work on Linode. I am using compilation code of conduct and parameters given in the documentation: cmake .. -DXMRIG_DEPS=scripts/deps -DBUILD_STATIC=ON

Long story short - I would like to compile XMRig to static executable that works on most Linux distributions (like the one that you can download from official XMRig website). Inb4 I have NOT made any changes to the source code yet - first I am trying to figure out the proper compilation method.

Any ideas?
